osu.Game/Skinning/DefaultBeatmapSkin.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Skinning
+{
+    public class DefaultBeatmapSkin : DefaultSkin, IBeatmapSkin
```

Comment:
not sure I see the point, but fine, I guess

Reply:
WorkingBeatmap uses a default skin class, this is why it exists, see:
https://github.com/ppy/osu/pull/10003/files#diff-de69e6e0a80b6ab97364a0919bf10d2aR280

Reply:
yes, I did read the entire diff. what I'm trying to say is that I would have probably just stuck `IBeatmapSkin` on `DefaultSkin` instead of doing this, but it's probably not worth splitting hairs over.